UI/Web/src/app/book-reader/_services/book.service.ts

```diff
@@ -28,7 +28,7 @@ export class BookService {
   getFontFamilies(): Array<FontFamily> {
     return [{title: 'default', family: 'default'}, {title: 'EBGaramond', family: 'EBGaramond'}, {title: 'Fira Sans', family: 'Fira_Sans'},
     {title: 'Lato', family: 'Lato'}, {title: 'Libre Baskerville', family: 'Libre_Baskerville'}, {title: 'Merriweather', family: 'Merriweather'},
-    {title: 'Nanum Gothic', family: 'Nanum_Gothic'}, {title: 'RocknRoll One', family: 'RocknRoll_One'}, {title: 'Open Dyslexic', family: 'OpenDyslexic2'}];
+    {title: 'Nanum Gothic', family: 'Nanum_Gothic'}, {title: 'Open Dyslexic', family: 'OpenDyslexic2'}, {title: 'RocknRoll One', family: 'RocknRoll_One'}, {title: 'Fast Font Serif', family: 'FastFontSerif'}, {title: 'Fast Font Sans', family: 'FastFontSans'}];
```

Comment:
Title should say something about Bionic, otherwise it won't be apparent.

Reply:
Resolved in [948bce6](https://github.com/Kareadita/Kavita/pull/3512/commits/948bce67acf67e77d2cbe3065ee957da714ad6bf)